Firmware/mmu2.cpp

```diff
@@ -601,6 +610,27 @@ void MMU2::manage_response(const bool move_axes, const bool turn_off_nozzle) {
         manage_inactivity(true); // calls LogicStep() and remembers its return status
         lcd_update(0);
 
+        if (mmu_print_saved & SavedState::CooldownPending)
+        {
+            if (!nozzleTimeout.running())
+            {
+                nozzleTimeout.start();
+                LogEchoEvent(" Cooling Timeout started");
```

Comment:
Approving PR as it looks good to me. Maybe remove the space in front from the log? :)

Reply:
I'm anticipating we'll remove some these log messages once we don't need them for debugging/testing anymore since they take up valuable storage space otherwise... :)